main.go

```diff
@@ -282,6 +281,7 @@ func (c *nvmeCollector) makeMetric(description *prometheus.Desc, valType prometh
 	value := gjson.Get(result, substring).Float()
 	if strings.Contains(substring, "temperature") {
 		// Leave it alone, if it's in Kelvin
+		if *c.temperatureScale == "celsius" {
```

Comment:
looks like you had a weird merge conflict here?

Reply:
I don't think so...that looks good...

Reply:
see the line below